php/example_code/dynamodb/Readme.md

```diff
@@ -0,0 +1,71 @@
+# Amazon DynamoDB basics
```

Comment:
Compare to the [template in the wiki](https://github.com/awsdocs/aws-doc-sdk-examples/wiki/Service-level-README-template).
* Some headings are different (main title, purpose -> overview).
* Missing single-action code example list.
* Additional resources list.

Reply:
Reworked using the template